bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/tHbj_TL_M1000GeV_W10p/tHbj_TL_M1000GeV_W10p_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+set group_subprocesses Auto
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+ ta+ 
```

Comment:
probably the last thing: all the definitions are not needed, right? If so please cleanup
define l+ = e+ mu+ ta+ 
define l- = e- mu- ta-
define vl = ve vm vt
define vl~ = ve~ vm~ vt~
define bb = b b~
define tt = t t~
define ww = w+ w-

Reply:
We can delete unused lines, but do we really need to redo the gridpacks since we define a 4FS proton in the card directly. 

define p = g u c d s u~ c~ d~ s~
define j = g u c d s u~ c~ d~ s~

Reply:
I would say so, as discussed in https://github.com/cms-sw/genproductions/pull/1264

Reply:
Please see #1408 as a new PR with all the cleanups. I have issues with MG5 for THtj_RH samples when producing gridpacks, but I think the card files are all fine. See https://hypernews.cern.ch/HyperNews/CMS/get/generators/3634/1.html

Reply:
sorry why a new PR? can you merge that branch with this one?
In this way I can see only the last modifications...

Reply:
Because merging is very difficult, unless Devdatta rebase his branch. Please see below.
https://github.com/dmajumder/genproductions/compare/SingleTtHWide_14TeV...skhalil:TtotH_14TeV?expand=1
The easiest way is the Devdatta close this request and you accept mine #1408